index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+       <h1 class="name">John Doe</h1>
+        <nav class="hamburger"> 
+            <div></div>
+            <div></div>
+            <div></div>
```

Comment:
Please use the `hamburger` icon image available in Figma Template instead of `<div>` tags.